### DISCUSSION TITLE
How to add a parameter such as locate="Paris,France"?

### DISCUSSION THREAD

**Discussion by wsw70** [go-resty/resty#647] [Q&A]
Summary: I would like to add **verbatim** `locate="Paris,France"` as a parameter to a GET call

Consider the following code:

```go
town := "Paris"
country := "France"
req := resty.New().R().
	SetHeader("Accept", "application/json").
	SetResult(response).
	SetQueryParams(map[string]string{
		"locate": fmt.Sprintf("\"%s,%s\"", town, country),
		"json":   "1",
		"auth":   "XXX",
	})
resp, err := req.Get("https://geocode.xyz")
```

My intent was to have the following request: `https://geocode.xyz?auth=XXX&json=1&locate="Paris,France"`  but the URI that is actually sent is `https://geocode.xyz?auth=XXX&json=1&locate=%22Paris%2CFrance%22`.

I did all kinds of attempts to force the quotes and comma in the call, but they always get encoded. Can I use `resty` to make such a call?

**Comment by jeevatkm** [comment]
@wsw70 Have you tried this `https://geocode.xyz?auth=XXX&json=1&locate="Paris,France"` with Go's default HTTP client?
